save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -146,7 +147,12 @@ class WarnPlugin(
             return failTestResult(originalPaths, ex, execCmd)
         }
 
-        val actualWarningsMap = collectActualWarningsWithLineNumbers(result, warnPluginConfig)
+        val actualWarningsMap = if (warnPluginConfig.actualWarningsFileName != null) {
```

Comment:
may be `let` here (as suggested in diktat)?

Reply:
Yes, fixed